R/pkg/R/functions.R

```diff
@@ -1906,6 +1906,7 @@ setMethod("atan2", signature(y = "Column"),
 
 #' @details
 #' \code{datediff}: Returns the number of days from \code{y} to \code{x}.
+#' If \code{y} is later than \code{x} then the result is positive.
```

Comment:
I'd add the same sentence to Python and Scala side too.

Reply:
Done